pkg/install/install.go

```diff
@@ -543,6 +557,41 @@ func replaceImage(image string, objs []kclient.Object) ([]kclient.Object, error)
 	return objs, nil
 }
 
+func replaceCabundleVolumes(objs []kclient.Object) ([]kclient.Object, error) {
```

Comment:
```suggestion
func replaceCABundleVolumes(objs []kclient.Object) ([]kclient.Object, error) {
```

Reply:
@thedadams Good question. In here we actually replace the whole CA bundle so there is no need to re-run `update-ca-certificates`. 

Also all the controller runs as user 1000 so we can't really touch the system cert dir, in the future we might allow user to mount additional cacerts and we can figure out a way to combine the certs with system default.